warden/x/async/keeper/abci.go

```
@@ -201,6 +226,20 @@ func (k Keeper) processVE(ctx sdk.Context, fromAddr []byte, ve types.AsyncVoteEx
 		}
 	}
 
+	log := ctx.Logger().With("module", "x/async")
+
+	if ve.UpdateHandlers {
+		if err := k.ClearHandlers(ctx, fromAddr); err != nil {
+			log.Error("clear handlers: %w", err)
```

if something goes wrong we should return the error, not just log it